homeassistant/components/shelly/sensor.py

```diff
@@ -142,12 +146,30 @@
     ("sensor", "tilt"): BlockAttributeDescription(name="tilt", unit=DEGREE),
 }
 
+REST_SENSORS = {
+    "rssi": RestAttributeDescription(
+        name="RSSI",
+        unit=SIGNAL_STRENGTH_DECIBELS,
+        device_class=sensor.DEVICE_CLASS_SIGNAL_STRENGTH,
+        default_enabled=False,
+        path="wifi_sta/rssi",
+    ),
+    "uptime": RestAttributeDescription(
```

Comment:
This needs to be a timestamp of when the device was booted. Relative times that update on each update are not allowed.

Reply:
Correct, is converted from relative to timestamp in utils.py:

```
if device_class == DEVICE_CLASS_TIMESTAMP:
        last_boot = datetime.utcnow() - timedelta(seconds=_attribute_value)
        _attribute_value = last_boot.replace(microsecond=0).isoformat()
```